Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
```

Comment:
+1 I will take this statement as is :  op-version was introduced to ensure gluster bits running with different version of bits do not end up in a problem and backward compatibility issues can be tackled.